extensions/amp-next-page/amp-next-page.md

```diff
@@ -51,19 +51,19 @@ limitations under the License.
 
 ## Behavior
 
-Given a list of recommended documents, `amp-document-recommendations` tries to load them after the current document, provding an infinite-scroll type experience.
+Given a list of recommended documents, `amp-next-page` tries to load them after the current document, providing an infinite-scroll type experience.
```

Comment:
should `recommended documents` become just `pages`?

Reply:
Done.